test/suite/test_hs09.py

```diff
@@ -43,7 +43,7 @@ class test_hs09(wttest.WiredTigerTestCase):
     uri = "table:test_hs09"
     key_format_values = [
         # The commented columnar tests needs to be enabled once columnar page instantiated is fixed in (WT-6061).
-        #('column', dict(key_format='r')),
+        # ('column', dict(key_format='r')),
```

Comment:
Well, actually I ran hs09 with only `test_prepared_updates_not_written_to_hs` disabled, and it runs fine:

```
diff --git a/test/suite/test_hs09.py b/test/suite/test_hs09.py
index d9943b29f..8e0588e30 100644
--- a/test/suite/test_hs09.py
+++ b/test/suite/test_hs09.py
@@ -43,7 +43,7 @@ class test_hs09(wttest.WiredTigerTestCase):
     uri = "table:test_hs09"
     key_format_values = [
         # The commented columnar tests needs to be enabled once columnar page instantiated is fixed in (WT-6061).
-        # ('column', dict(key_format='r')),
+        ('column', dict(key_format='r')),
         ('integer', dict(key_format='i')),
         ('string', dict(key_format='S')),
     ]
@@ -108,6 +108,9 @@ class test_hs09(wttest.WiredTigerTestCase):
         self.check_ckpt_hs(value2, value1, 2, 3)
 
     def test_prepared_updates_not_written_to_hs(self):
+        if self.key_format == 'r':
+            return
+
         # Create a small table.
         create_params = 'key_format={},value_format=S'.format(self.key_format)
         self.session.create(self.uri, create_params)
```

Reply:
We disabled this because the current branch was supposed to be for tests that run successfully in full. We are making the changes to hs09 elsewhere (enabling column store and disabling the prepared update).

Reply:
hs06 has similar disable just prepare portions, hence I thought we are enabling as much as we can